.gitignore

```diff
@@ -1,3 +1,4 @@
 .idea
 .coverage
 *__pycache__*
+.DS_Store
```

Comment:
Dustin also changed this file, so we will want to ensure that this also gets added